### PR TITLE
feat: Request service lab

### PR DESCRIPTION
**Pull request by bukandicki**
### JIRA Link
https://blocksphere2020.atlassian.net/browse/DBIO-851

### Changelog / Description
Update request service flow lab

### CODE REVIEW

File: src/views/Dashboard/Lab/Services/Add.vue [hunk 1]
```diff
@@ -452,47 +463,48 @@ export default {
...
+          this.wallet,
+          {
+            name: this.name,
+            pricesByCurrency: [
+              {
+                currency: this.currencyType,
+                totalPrice: await toEther(this.price + this.qcPrice),
+                priceComponents: [
+                  {
+                    component: "test_price",
```
Comment (hilyds) [inline src/views/Dashboard/Lab/Services/Add.vue:479]: should re-check if this is test_price or testing_price?

**Review by hilyds — COMMENTED** [review]
please PR to branch `main` instead of `development`. besides that, all looks good to me.

File: src/views/Dashboard/Lab/Services/Add.vue (hunk 1 above)
Comment (bukandicki) [reply src/views/Dashboard/Lab/Services/Add.vue:483]: Hi mbak @hilyds, I've tried to double-check, mostly on the backend they use `testing_price`, and for `test_price` I can't find any. Is it using `testing_price`? please check my cross-check video below 👇🏼 

https://www.awesomescreenshot.com/video/6158148?key=89324edf962cc84d6b403c7b79ecb641

**Review by hilyds — APPROVED** [review]
LGTM

**Pull request merged**